tox.ini

```diff
@@ -109,37 +108,3 @@ commands =
     {[testenv:integration]commands}
     {[testenv]covreport}
 
-#######################################################################################
-# Software Package Build & Release
-#######################################################################################
-[testenv:build]
-description = Prepare Python source and binary packages for release.
-basepython = python3
-skip_install = false
-extras =
-    dev
-commands =
-    bash -c 'rm -rf build/* dist/*'
-    python -m build
-
-[testenv:testrelease]
-description = Do a dry run of Python package release using the PyPI test server.
-basepython = python3
-skip_install = false
-extras =
-    dev
-commands =
-    {[testenv:build]commands}
-    twine check dist/*
-    twine upload --verbose --repository testpypi --skip-existing dist/*
-
-[testenv:release]
-description = Release the package to the production PyPI server.
-basepython = python3
-skip_install = false
-extras =
-    dev
-commands =
-    {[testenv:build]commands}
-    twine check dist/*
-    twine upload --verbose --skip-existing dist/*
```

Comment:
Removed because it takes place in the `release` workflow now.

Reply:
wooohoo!